### ISSUE TITLE
A11y_Visual Studio Code Services_Docs_Left Right Navigation_Screenreader : Narrator doesn't announce the "Expand/Collapse" status for dropdown in "Left navigation" links.

### ISSUE THREAD

**Issue by jadhavprathmesh01** [microsoft/vscode-docs#5106]
**GitHubTags:**

#A11yMAS; #A11yTCS; #A11ySev2; #DesktopWeb; #Visual Studio Code Services; #BM-Visual Studio Code Services-Web-Jan2022; #Web; #Narrator; #NVDA; #WCAG1.3.1; #FTP; #STP; #ChromiumEdge; #Chrome;

**Environment Details:**
Application Name: Visual Studio Code Services
Windows Version: Windows 11

**Repro Steps:**

1. Launch the application Visual Studio Code Services
2. Tab navigate to "Docs" link.
3. Left and right navigation will appear.
4. Turn on the narrator.
5. Tab navigate to the Left navigation links and observe that the narrator is announcing or no "Expand/Collapse" status for dropdown.

**Actual:**
The narrator is not announcing the "Expand/Collapse" status for dropdown when we navigate using keyboard.
**NOTE:** This issue is not reproducing in NVDA screenreader.

**Expected Result:**
The narrator should announce the "Expand/Collapse" status for dropdown.

**Narrator attachment:**
![A11y_Visual Studio Code Services_Docs_Left Right Navigation_Screenreader](https://user-images.githubusercontent.com/87858881/149924846-1118faaf-0d61-41ed-aca8-64c42857345a.gif)

**NVDA Screenreader attachment:**

![A11y_Visual Studio Code Services_Docs_Left Right Navigation_Screenreader](https://user-images.githubusercontent.com/87858881/149928610-a5bf96a6-8c9d-44f8-8960-b350ed64637a.png)

**Comment by rzhao271** [comment]
@jadhavprathmesh01 can you give an example of a site where Narrator reads the expanded/collapsed item properly? I tried looking through some other sites using Windows 10 with Microsoft Edge and Narrator, but couldn't find an example that worked.

**Comment by jadhavprathmesh01** [comment]
@gregvanl sorry for the delay. I have created a new bug https://github.com/microsoft/vscode-docs/issues/5203. Also this link https://visualstudio.microsoft.com/ is an example where we can find Narrator announce collapsed status. Please check the **Download Visual Studio** dropdown where the narrator is announcing collapsed status.

**Comment by sspatil123** [comment]
GithubTags:#Closed;Version 1.75
![image](https://user-images.githubusercontent.com/85863999/217259211-c8d85923-947b-4112-b1b9-08e674f8b14c.png)
